### PR TITLE
dada2: add dada2 workflow for paired end data

### PR DESCRIPTION
**Pull request by bernt-matthias**

### CODE REVIEW

**Comment by bernt-matthias** [comment]
separated from https://github.com/galaxyproject/tools-iuc/pull/2483

**Comment by yvanlebras** [comment]
Thank you Matthias ! Really important as it is particularly hard to manage paired reads with dada2! THANK YOU for all your amazing work on these complicated toolsuites!

File: workflows/amplicon/dada2/dada2_paired.ga [hunk 1]
```diff
@@ -0,0 +1,823 @@
...
+            "name": "Matthias Bernt"
+        },
+        {
+            "class": "Organization",
+            "name": "UFZ Leipzig"
+        }
+    ],
+    "format-version": "0.1",
+    "license": "MIT",
+    "name": "dada - paired",
```
[the remark this answers is not in the document]
Comment (mvdbeek) [reply workflows/amplicon/dada2/dada2_paired.ga:17]: Did you miss this one ? Without the release tag we won't know what version to create for the github release.

Comment (mvdbeek) [inline workflows/amplicon/dada2/dada2_paired.ga:17]: Actually, do you maybe a more descriptive name for the workflow ?

File: workflows/amplicon/dada2/dada2_paired.ga [hunk 2]
```diff
@@ -0,0 +1,824 @@
...
```
Comment (bernt-matthias) [inline workflows/amplicon/dada2/dada2_paired.ga:17]:
```suggestion
    "name": "dada2 amplicon analysis pipeline - for paired end data",
```

**Review by mvdbeek — APPROVED** [review]

**Comment by mvdbeek** [comment]
Awesome, thanks a lot @bernt-matthias!

**Pull request merged**